.github/workflows/haskell.yml

```diff
@@ -111,18 +111,19 @@ jobs:
       run: cabal check
     - name: Create source distribution file (cabal)
       run: cabal sdist
-    - name: Identify source distribution file (cabal)
+    - name: Identify atlas version (cabal)
       run: |
-        export SRC_TGZ=$(cabal info . | awk '{print $2 ".tar.gz";exit}') ;
-        echo "SRC_TGZ: $SRC_TGZ"
+        export ATLAS_VERSION=$(cabal info . | awk '{print $2 ;exit}') ;
+        echo "ATLAS_VERSION: $ATLAS_VERSION"
+        echo "ATLAS_VERSION=${ATLAS_VERSION}" >> $GITHUB_ENV
```

Comment:
This line is necessary to make the ATLAS_VERSION env var available in the later steps.

For further details see the official GitHub documentation:
 - (GitHub Actions Documentation: Workflow commands - Setting an environment variable)[https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable]